src/en/space-station-14/round-flow/proposals/ninja.md

```diff
@@ -0,0 +1,94 @@
+
+- Steal X spesos - A strongbox for Cargo containing all their deposited spesos will be located in the vault, the Space Ninja is tasked with stealing a designated amount - An alternative to "Steal X technologies".
+- Upload the law board - An AI law board with a Spider Clan-oriented law set, the Space Ninja is tasked with uploading it to the AI upload console - An alternative to "Detonate your spider charge".
+- Hack Si-XXXX - The Space Ninja's kill-target equivalent, they are tasked with hacking a specific cyborg, turning it into a zealous Spider Clan tech-priest that obsesses over hoarding all manners of electronics and machines.
+
+## Objective Changes
+
+- All stealing objectives can now only be performed with a specialized item that is dropped by the ninja upon death, unlike previously where they only used their gloves - This allows the crew to regain their losses.
+- Make an announcement - The communications computer objective now revolves around the Space Ninja sending a Spider Clan Announcement from it, the contents of said announcement being up to their discretion. Like before, it requires the use of turned on space ninja gloves, the one key difference being that now the EMAG interaction is instantaneous, overriding all access restrictions as well as unlocking Spider Clan as an announcement sender. (Replaces "Call in a threat", the new objective keeping the threat-calling functionality is still in discussion)
+- Hack X - 1-3 of the Miscellaneous Interactions machines is picked to be hacked by the Space Ninja. Note that you can still hack machines that aren't included in your objectives. (Replaces "Set Everyone to Wanted")
+
+## Miscellaneous Interactions
+
+- Hacking the criminal records computer is now one of the many miscellaneous actions the Space Ninja can do in order to frustrate the crew - The error announcement will now only be sent by the criminal records computer on the Security channel, opposed to the previous station-wide Central Command Announcement.
+- The crew monitoring server is now prone to being hacked by the Space Ninja, doing so will disable everyone's sensors until deconstructed and reconstructed.
+- The station records computer is now prone to being hacked by the Space Ninja, doing so will delete a small % of random personnel from it accompanied by a station-wide announcement.
+- The camera router is now prone to being hacked by the Space Ninja, doing so will kill a large % of its connected cameras and render them useless.
+
+# Balance Changes
+
+Changes mostly geared towards fixing previously outright broken mechanics, as well to accomodate the new additions.
+
+## Nerfs
+
+These nerfs are intended to make previous go-to combat tactics less prevalent, hence incentivizing a more varied playstyle.
+
+- [Fix ninja stunlocking](https://github.com/space-wizards/space-station-14/pull/33244)
+- All Space Ninjas are now bio-engineered humans, making it so they don't get specie-specific attributes such as tail-dragging.
```

Comment:
Having to restrict an antags species to human is lame and circumvents a whole part of the game: different species. It's part of this games character.

See: SS13 where nukies were only humans because the AI was speciesist. We can do better in SS14.

Reply:
> Having to restrict an antags species to human is lame and circumvents a whole part of the game: different species. It's part of this games character.
> 
> See: SS13 where nukies were only humans because the AI was speciesist. We can do better in SS14.

I had a relatively discussion about this on discord recently (after the comments one), and really I think I'll be implementing Dragonjspider's "The role needs to randomize your character when spawned entirely" proposal, as it'd be the best of both worlds imho, having it be your character could prove not nearly as fun as if you had done it with a different species, but having always be fixed to a certain species could prove to be against the game's character, and so it'll pick a species from a certain species pool (eg. humans, dwarves, reptillians, arachnids - as they are moderately "standard")